spec.emu

```diff
@@ -502,54 +1375,158 @@ contributors: Luca Casonato, Guy Bedford, Nicolò Ribaudo
 
             <emu-alg>
               1. Assert: _state_.[[IsLoading]] is *true*.
-              1. <ins>Let _importOnlyAsPhase_ be *false*.</ins>
-              1. <ins>If _referrer_ has been specified, then</ins>
-                1. <ins>Assert: _specifier_ has been specified.</ins>
```

Comment:
We should still use `_referrer_` and `_specifier_` to know if we are in a "source phase import" and thus stop recursively loading the dependencies. As the spec is written in this PR, we load all the dependencies of source imports.

Something like this:
```
1. Let _loadDependencies_ be *true*.
1. If _referrer_ has been specified, then:
  1. Assert: _specifier_ has been specified.
  1. Let _request_ be the only ModuleRequest Record in _referrer_.[[RequestedModules]] such that _request_.[[Specifier]] is _specifier_.
  1. If _request_.[[Phase]] is ~source~, set _loadDependencies_ to *false*.
```

We still need the changes to `ContinueModuleLoading` to propagate referrer and specifier.